File: libs/common/models/vw-pmp-psipspecialization-yearly-cost.ts



export class VwPmpPsipspecializationYearlyCost {
    public specializationId: number = 0;
    public sessionOfStudyId: number = 0;
    public sessionStartDate: Date;
    public sessionEndDate: Date;
    public cost: number;

    constructor(init? : Partial<VwPmpPsipspecializationYearlyCost>) {
        
        Object.assign(this, init);
    }
}

File: apps/institution-api/Models/EntityFramework/VwPmpPsipspecializationYearlyCost.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models.EntityFramework
{
    public partial class VwPmpPsipspecializationYearlyCost
    {
        public int SpecializationId { get; set; }
        public int SessionOfStudyId { get; set; }
        public DateTime? SessionStartDate { get; set; }
        public DateTime? SessionEndDate { get; set; }
        public decimal? Cost { get; set; }
    }
}
